capabilityreceiver/pkg/capabilityhandler/capabilityreceiver_test.go

```diff
@@ -20,6 +20,7 @@ var testQueueMsg = map[string]interface{}{
 	"tlsVersion":        "TLS 1.2",
 	"smarthttpResponse": 0,
 	"smartResp":         nil,
+	"responseTime":      0.1234,
```

Comment:
I would expect to see some sort of assertion somewhere in this test file that verifies that this value was received correctly, as is done with all of the other values

Reply:
This should be all set now!